.github/ISSUE_TEMPLATE/gap.yml

```diff
@@ -57,4 +57,5 @@ body:
 description: >
   Use this template to report documentation gaps: information that is
   missing altogether, or that is incomplete.
+labels: gap
```

Comment:
```suggestion
label: gap
```

Reply:
Again, [nope](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#top-level-syntax), sorry. :)